src/App.js

```diff
@@ -1,25 +1,63 @@
-import logo from './logo.svg';
+import React, { Component } from 'react'
 import './App.css';
+import PokeList from './PokeList.js'
+import Dropdown from './Dropdown.js';
+class App extends Component {
+  state = { 
+    data: [],
+    loading: true,
+    query: null,
```

Comment:
The default data you set in your state should always reflect the same data TYPE that your final result will be. 

In this case, your query would be an empty string rather than null since your eventual data will be a string type.